_posts/2017/2017-11-26-choose-uuids-for-model-ids-in-rails.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Choose UUIDs for model ids in Rails'
+description: 'A little more complexity, a little more security'
+layout: article
+category: ruby
+image:
+  base: '2017/choose-uuids-for-model-ids-in-rails'
+  alt: 'Rainbow UUIDs'
+---
+
+Ruby on Rails has had the [ability](https://github.com/rails/rails/pull/21762) to use UUIDs ([Universally Unique IDentifiers](https://en.wikipedia.org/wiki/Universally_unique_identifier)) for ActiveRecord models since version 5.0.
+
+These are a native column type in PostgreSQL, there's more details on native Postgres types in the [Rails Guides](http://guides.rubyonrails.org/active_record_postgresql.html).
+
+## Instead of…
+
+…using Rails default incrementing integer IDs.
```

Comment:
* Rails' ? (possessive)

Reply:
I would finish this with '...' as opposed to a full-stop, because the sentence is still continuing.